pkg/gitreceive/build.go

```diff
@@ -274,3 +272,26 @@ func prettyPrintJSON(data interface{}) (string, error) {
 	}
 	return string(formatted.Bytes()), nil
 }
+
+func getProcFile(getter storage.ObjectGetter, dirName, bucketName, procfileKey string) (pkg.ProcessType, error) {
+	procType := pkg.ProcessType{}
+	if _, err := os.Stat(fmt.Sprintf("%s/Procfile", dirName)); err == nil {
+		rawProcFile, err := ioutil.ReadFile(fmt.Sprintf("%s/Procfile", dirName))
+		if err != nil {
+			return nil, fmt.Errorf("reading %s/Procfile", dirName)
```

Comment:
@kmala can you please include the original error at the end of this error string (as you do below)?